docs/docs/how-tos/sequence.ipynb

```diff
@@ -0,0 +1,327 @@
+   "source": [
+    "### Define nodes\n",
+    "\n",
+    "Our [nodes](../../concepts/low_level/#nodes) are just Python functions that read our graph's state and make updates to it. The first argument to this function will always be the state:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def step_1(state: State):\n",
+    "    return {\"value_1\": \"a\"}\n",
+    "\n",
+    "\n",
+    "def step_2(state: State):\n",
+    "    current_value_1 = state[\"value_1\"]\n",
+    "    return {\"value_1\": f\"{current_value_1} b\"}\n",
+    "\n",
+    "\n",
+    "def step_3(state: State):\n",
+    "    return {\"value_2\": 10}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that when issuing updates to the state, each node can just specify the value of the key it wishes to update.\n",
+    "\n",
+    "By default, this will **overwrite** the value of the corresponding key. You can also use [reducers](../../concepts/low_level/#reducers) to control how updates are processed-- for example, you can append successive updates to a key instead. See [this guide](../../how-tos/state-reducers) for more detail.\n",
+    "\n",
+    "### Define graph\n",
+    "\n",
+    "We use [StateGraph](../../concepts/low_level/#stategraph) to define a graph that operates on this state.\n",
+    "\n",
+    "We will then use [add_node](../../concepts/low_level/#messagesstate) and [add_edge](../../concepts/low_level/#edges) to populate our graph and define its control flow."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langgraph.graph import StateGraph\n",
+    "\n",
+    "graph_builder = StateGraph(State)\n",
+    "\n",
+    "# Add nodes\n",
+    "graph_builder.add_node(step_1)\n",
+    "graph_builder.add_node(step_2)\n",
+    "graph_builder.add_node(step_3)\n",
+    "\n",
+    "# Add edges\n",
+    "graph_builder.set_entry_point(\"step_1\")\n",
+    "graph_builder.add_edge(\"step_1\", \"step_2\")\n",
+    "graph_builder.add_edge(\"step_2\", \"step_3\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "!!! tip \"Specifying custom names\"\n",
+    "\n",
+    "    You can specify custom names for nodes using `.add_node`:\n",
+    "\n",
+    "    ```python\n",
+    "    graph_builder.add_node(\"my_node\", step_1)\n",
+    "    ```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that:\n",
+    "\n",
+    "1. `.add_edge` takes the names of nodes, which for functions defaults to `node.__name__`.\n",
+    "2. We must specify the entry point of the graph. For this we used `.set_entry_point`, but we can also control where our graph starts and ends by adding edges with the [START node](../../concepts/low_level/#start-node) and [END node](../../concepts/low_level/#end-node).\n",
```

Comment:
let's change this to just recommend `.add_edge()` instead of entrypoint
also, let's remove the part about "controlling where the graph ends w/ END" cause it confuses people. in fact, it's good to clarify that here as well for new folks, e.g. add something like "the graph halts when there are no more tasks to execute"